<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInfoDialogClass</class>
 <widget class="QWidget" name="AddInfoDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddInfo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>StudentInfo</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="lblID">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditID"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblClass">
            <property name="text">
             <string>Class</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditClass"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblTerm">
            <property name="text">
             <string>Term</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditTerm"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GradeInfo</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblID_6">
            <property name="text">
             <string>Chinese</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblID_7">
            <property name="text">
             <string>Math</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxChinese"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxMath"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblID_8">
            <property name="text">
             <string>English</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxEnglish"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblID_9">
            <property name="text">
             <string>Computer</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxComputer"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
